lib/Cro/WebSocket/Client.pm6

```diff
@@ -54,12 +61,9 @@ class Cro::WebSocket::Client {
             my $magic = "258EAFA5-E914-47DA-95CA-C5AB0DC85B11";
             my $answer = encode-base64(sha1($key ~ $magic), :str);
 
-            my %options = headers => (Cro::HTTP::Header.new(name => 'Upgrade', value => 'websocket'),
-                Cro::HTTP::Header.new(name => 'Connection', value => 'Upgrade'),
-                Cro::HTTP::Header.new(name => 'Sec-WebSocket-Version', value => '13'),
-                Cro::HTTP::Header.new(name => 'Sec-WebSocket-Key', value => $key),
-                Cro::HTTP::Header.new(name => 'Sec-WebSocket-Protocol', value => 'echo-protocol'));
+            @!headers.push: Cro::HTTP::Header.new(name => 'Sec-WebSocket-Key', value => $key);
```

Comment:
This will add the `Sec-WebSocket-Key` to `@!headers` for each request, meaning that multiple connections made with the same client object will send duplicates of that header. Make a local copy (`my @headers = @!headers`) and then push it onto that instead.

Reply:
OK, I did it so.